src/tailwind.scroll-shadows.js

```diff
@@ -0,0 +1,62 @@
+import plugin from 'tailwindcss/plugin.js';
+
+/**
+ * Utility class to show scroll-hint shadows at the top and bottom of a
+ * scrollable frame element if:
+ *  - The content height exceeds the frame height: i.e. can be scrolled, and
+ *  - The content is scrollable in the shadow's direction (up or down)
+ *
+ * Shadows are not visible once the frame has been scrolled all the way in the
```

Comment:
This may be copied and pasted from SASS, but such nice documentation ❤️

Reply:
Thanks! This CSS is frankly complicated and non-obvious, so comments were needed.